src/Main.java

```diff
@@ -1,7 +1,34 @@
 public class Main {
     public static void main(String[] args) {
 
-        // Тут має розгортатися епічна битва між Супергероями
+        Superhero batman = new Superhero();
+        Superhero superman = new Superhero();
 
+        batman.name = "Batman";
+        batman.attackPower = 40;
+        batman.defensePower = 25;
+
+        superman.name = "Superman";
+        superman.attackPower = 45;
+        superman.defensePower = 15;
+
+        batman.attack(superman);
```

Comment:
The only one thing you could do better -> why not use a `while` loop to make them fight without using so many lines of code?

They would fight until one of them dies, right, sounds like a place for a while loop